pom.xml

```diff
@@ -36,6 +36,11 @@
 		    <artifactId>json</artifactId>
```

Comment:
Can we remove the keycloak dependency now? I can't add a comment on the specific line but it's like 10 lines up from here